.github/workflows/PR.yml

```diff
@@ -68,7 +68,7 @@ jobs:
       with:
          runs-on: ubuntu-latest
          ref: ${{ inputs.ref }}
-         task: ${{ matrix.target }}
+         task: ${{ matrix.target }} --info
```

Comment:
TODO: remove `--info` once build is successful

Reply:
Did the trick, it seems. Got past the `extensions-http` test. Yay!

Reply:
```suggestion
         task: ${{ matrix.target }}
```